akka-remote/akka-profiling-hifreq.jfc

```diff
@@ -0,0 +1,824 @@
+          <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+          <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+          <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+        </event>
+
+        <event name="akka.remote.artery.jfr.TransportUniqueAddressSet" label="Unique address set">
+          <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+          <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+          <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+        </event>
+
+        <event name="akka.remote.artery.jfr.TransportStarted" label="Transport started">
+          <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+          <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+          <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+        </event>
+
+        <event name="akka.remote.artery.jfr.TransportMaterializerStarted" label="Materializer started">
+          <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+          <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+          <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+        </event>
+
+      </category>
+
+      <category label="Tcp">
+
+        <category label="Inbound">
+
+          <event name="akka.remote.artery.jfr.TcpInboundBound" label="Bound">
+            <setting name="stackTrace" label="Stack Trace" description="Record stack traces" contentType="jdk.jfr.Flag">false</setting>
+            <setting name="enabled" label="Enabled" description="Record event" contentType="jdk.jfr.Flag">true</setting>
+            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
+          </event>
+
+          <event name="akka.remote.artery.jfr.TcpInboundConnected" label="Connected">
+            <setting name="threshold" label="Threshold" description="Record event with duration above or equal to threshold" contentType="jdk.jfr.Timespan">0 ns</setting>
```

Comment:
can we skip the threshold since we don't use the `duration` measurement in any events

Reply:
Decided to remove the sample jfc entirely and instead tell users to create their own as this contains explicit settings like this that could potentially change name over time etc.